src/hooks/useDetectMobile.js

```diff
@@ -11,5 +11,5 @@ export default function useDetectMobile() {
             window.removeEventListener('resize', handleResize)
         }
     }, [])
-    return width < 768
+    return width < 1180
```

Comment:
did you mean to edit the function for useDetectMobileOrTablet?

Reply:
Yes ! it's now edited :)